llvm/test/CodeGen/NVPTX/cp-async-bulk-tensor-g2s.ll

```diff
@@ -0,0 +1,169 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90 -mattr=+ptx80| FileCheck --check-prefixes=CHECK_PTX64 %s
```

Comment:
It may be useful to convert the test to generate the checks automatically.
While we're dealing with direct intrinsic to instruction mapping here, and the manual checks work well enough for that, in this case we're dealing with a lot of arguments with similar types and it would be good to verify that those are passed in correct order.

As things are, if we accidentally swap some arguments the tests may still pass, as the code would still be syntactically valid for both LLVM and ptxas.

Reply:
Thanks for this suggestion! It would be very beneficial indeed.
Please let me know if the latest revision aligns with what you had in mind.